### PR TITLE
Add some missing docs

### PR DESCRIPTION
**Pull request by abarth**
These docs complete the dartdocs for their respective libraries.

### CODE REVIEW

**Comment by abarth** [comment]
@chinmaygarde

File: packages/flutter/lib/src/cassowary/solver.dart [hunk 1]
```diff
@@ -145,6 +145,10 @@ class _Row {
   }
 }
 
+/// Solves cassoway constraints.
```
Comment (yjbanov) [inline packages/flutter/lib/src/cassowary/solver.dart]: nit: cassowary

**Comment by yjbanov** [comment]
lgtm

File: packages/flutter/lib/src/cassowary/solver.dart [hunk 2]
```diff
@@ -153,6 +157,9 @@ class Solver {
   final List<_Symbol> _infeasibleRows = new List<_Symbol>();
   final _Row _objective = new _Row(0.0);
   _Row _artificial = new _Row(0.0);
+
+  /// A monotonically increasing value that indicates how many times the solver
+  /// has iterated.
```
Comment (Hixie) [inline packages/flutter/lib/src/cassowary/solver.dart]: should this be publicly mutable?

Comment (Hixie) [inline packages/flutter/lib/src/cassowary/solver.dart]: if so, the docs should mention the semantics of mutating it

File: packages/flutter_sprites/lib/src/sprite_box.dart [hunk 3]
```diff
@@ -85,6 +85,7 @@ class SpriteBox extends RenderBox {
   Duration _lastTimeStamp;
   double _frameRate = 0.0;
 
+  /// The number of frames per second this sprite box is producing.
```
Comment (Hixie) [inline packages/flutter_sprites/lib/src/sprite_box.dart]: is this instantaneous or averaged?

File: packages/flutter_test/lib/src/controller.dart [hunk 4]
```diff
@@ -17,8 +17,10 @@ import 'test_pointer.dart';
 ///
 /// For a variant of this class suited specifically for unit tests, see [WidgetTester].
 class WidgetController {
+  /// Creates a widget controller that uses the given binding.
   WidgetController(this.binding);
 
+  /// The binding instance used by the widget controller.
```
Comment (Hixie) [inline packages/flutter_test/lib/src/controller.dart]: there's only one binding instance, by definition. Maybe "A reference to the current instance of the binding" or something would make that clearer.

File: packages/flutter/lib/src/cassowary/solver.dart (hunk 2 above)
Comment (chinmaygarde) [inline packages/flutter/lib/src/cassowary/solver.dart]: The tick count is just a debugging aid that comes in handy when dumping solver state. We can get rid of it. Will attempt that now.

**Comment by chinmaygarde** [comment]
After this lands, I will get rid of the `tick` on the `_Symbol`.

**Pull request merged**